### PR TITLE
Create BuyerPickerFallbackCell

### PR DESCRIPTION
**Pull request by robinsalehjan**
# Why?

- In case there's no candidates up for review. We should inform the seller that in order for them to give a review, they need to be messaged by the buyer first.

# What?

- Implement `BuyerPickerFallbackCell`
- Add `func motorTransactionInviteCounterpartyViewDidSelectFallbackCell` to delegate protocol

# Show me

| Empty | With candidates |
| --- | --- |
| Simulator Screen Shot - iPhone 12 Pro - 2020-10-28 at 12 04 49 | Simulator Screen Shot - iPhone 12 Pro - 2020-10-28 at 12 07 54 |

### CODE REVIEW

File: FinniversKit/Sources/Fullscreen/Review/BuyerPickerViewModel.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Foundation
 public protocol BuyerPickerViewModel {
     var title: String { get }
     var profiles: [BuyerPickerProfileModel] { get }
+    var fallbackCell: BuyerPickerProfileModel { get set }
```
Comment (hakloev) [inline FinniversKit/Sources/Fullscreen/Review/BuyerPickerViewModel.swift:10]: Does it have to be `{ get set }`?

File: FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }()
+
+    lazy var hairlineView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.backgroundColor = .tableViewSeparator
+        return view
+    }()
+
+    lazy var chevronLabel = Label(style: .detail, withAutoLayout: true)
```
Comment (hakloev) [inline FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift:24]: Can we make these properties private?

File: FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    var model: BuyerPickerProfileModel? {
+        didSet {
+            titleLabel.text = model?.name ?? ""
+            chevronLabel.text = model?.chevronText ?? ""
```
Comment (hakloev) [inline FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift:29]: Wouldn't `x.text = model?.name` suffice? 🤔

File: FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        titleLabel.text = ""
```
Comment (hakloev) [inline FinniversKit/Sources/Fullscreen/Review/Cell/BuyerPickerFallbackCell.swift:44]: Same here, can it be `nil`?

**Review by hakloev — COMMENTED** [review]
LGTM, but had some questions :)

**Review by hakloev — APPROVED** [review]

**Comment by robinsalehjan** [comment]
Will wait with merging this until I got everything ready 👍

**Pull request merged**